docs/filesystem_layer/stratum1.md

```diff
@@ -87,14 +95,82 @@ You can request a firewall exception rule to be added for your Stratum 1 server
 
 Make sure to include the IP address of your server.
 
+## Step 3: Verification of the Stratum 1
+
+When the playbook has finished your Stratum 1 should be ready. In order to test your Stratum 1, even
+without a client installed, you can use curl.
+
+```bash
+curl --head http://url-to-your-stratum1/cvmfs/pilot.eessi-hpc.org/.cvmfspublished
+```
+This should return:
+
+```bash
+HTTP/1.1 200 OK
+...
+X-Cache: MISS from url-to-your-proxy
```

Comment:
This is actually the stratum 1's Squid proxy (might be different when you pass `--proxy <your-proxy>` to `curl` as well), so maybe we should use `url-to-your-stratum1` instead.

Reply:
I thought about that. We both use `url-to-your-stratum1` and `<S1_IP_>`. Should be maybe keep to one? or somehow catch both..

Reply:
Ah, yes, better to keep that consistent across the page.

Reply:
is `url-or-ip-to-your-stratum1` too much?

Reply:
Okay with me :-)